example/package.json

```diff
@@ -2,6 +2,10 @@
   "name": "webpack-rails-example",
   "version": "0.0.1",
   "license": "MIT",
+  "scripts": {
+    "compile":  "webpack -d --config config/webpack.config.js",
+    "serve": "webpack-dev-server -p --config config/webpack.config.js"
```

Comment:
When using `-p` option, I had the following issue: https://github.com/webpack/webpack/issues/1385 which was fixed by removing the `-p` option.

> when using new UglifyJsPlugin and --opimize-minimize (or -p) you are adding the UglifyJsPlugin twice. Omit the CLI option.